building/tracks/new/add-first-exercise.md

```diff
@@ -25,3 +22,52 @@ The "Hello, World!" exercise has some special rules applied to it:
 - The stub file contains an almost-working implementation, but instead of "Hello, World!" it uses "Goodbye, Mars!"
 - It has no `prerequisites`
 - It has no `practices`
+
+### Updating the config
+
+Start by adding an entry to the `exercises.practice` array in the top-level `config.json` file.
+
+```
+{
+  "exercises": {
+    "practice": [
+      {
+        "uuid": "",
+        "slug": "hello-world",
+        "name": "Hello World",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
+      }
+    ]
+  }
+}
+```
+
+You can use the [Configlet][configlet] tool to get a UUID.
+Download Configlet by running `bin/fetch-configlet` from the root of the repository.
+Then generate a UUID using the `bin/configlet uuid` command.
+
+### Generating required files
+
+To implement the "Hello, World!" exercise, you'll need to create several files.
+Which files exactly is described in the [Practice Exercises documentation](/docs/building/tracks/practice-exercises).
+
+Most of the files can be added automatically by running Configlet's `sync` command:
+
+```
+bin/configlet sync --update --docs --metadata --tests
+```
+
+In addition to the generated files, you will to create a test suite, a stub solution that serves as the starting point for the student, and a sample solution that CI uses to ensure that the test suite is in a good state.
```

Comment:
I feel like the bigger reason for having a sample solution is to prove that the exercise _can_ be solved. Maybe something like:

```suggestion
In addition to the generated files, you will to create a test suite, a stub solution that serves as the starting point for the student, and a sample solution that passes all the tests to verify it is possible to solve the exercise (CI will verify this).
```